spec/reek/configuration/configuration_file_finder_spec.rb

```diff
@@ -4,6 +4,8 @@
 require 'tmpdir'
 require_relative '../../spec_helper'
 
+include Reek::Configuration
+
```

Comment:
@chastell -> https://github.com/troessner/reek/issues/443

I believe we are all in favor of that.
That's why I wrote "one last time"..:)